src/project/internal/projectactionscontroller.cpp

```diff
@@ -624,7 +626,8 @@ void ProjectActionsController::clearRecentProjects()
 
 void ProjectActionsController::exportAudio()
 {
-    NOT_IMPLEMENTED;
+    IAudacityProjectPtr project = globalContext()->currentProject();
+    auto ret = interactive()->open(EXPORT_OPTIONS_URI);
```

Comment:
Since you’ve moved the export logic inside the export dialog, we can name the URI like EXPORT_URI.